### PR TITLE
darodrig.py

### PR DESCRIPTION
**Pull request by d-r-e**
First, errors are checked, so that there is just one parameter and it just contains alpha characters and spaces.
Then characters are printed by finding them on the dictionary. Then a new line is printed.

### CODE REVIEW

File: chall02/darodrig.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                    'C':'-.-.', 'D':'-..', 'E':'.', 
+                    'F':'..-.', 'G':'--.', 'H':'....', 
+                    'I':'..', 'J':'.---', 'K':'-.-', 
+                    'L':'.-..', 'M':'--', 'N':'-.', 
+                    'O':'---', 'P':'.--.', 'Q':'--.-', 
+                    'R':'.-.', 'S':'...', 'T':'-', 
+                    'U':'..-', 'V':'...-', 'W':'.--', 
+                    'X':'-..-', 'Y':'-.--', 'Z':'--..'}
+	if len(sys.argv) != 2 or sys.argv[1] == "":
+		print("usage: ./xlogin.py <a-zA-Z string>")
```
Comment (clafoutis42) [inline chall02/darodrig.py:15]: Your file is named `darodrig.py` and could be named differently too, therefore your usage is incorrect.  
You could have made it dynamic this way:
```suggestion
		print("usage: %s <a-zA-Z string>" % sys.argv[0])
```

File: chall02/darodrig.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+		return
+	for each in sys.argv[1]:
+		if each.isalpha() == 0 and each != ' ':
+			print("usage: ./xlogin.py <a-zA-Z string>")
+			return
```
Comment (clafoutis42) [inline chall02/darodrig.py:20]: Could have been written
```suggestion
        if not sys.argv[1].replace(" ", "").isalpha():
            return print("usage: %s <a-zA-Z string>" % sys.argv[0])
```

File: chall02/darodrig.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+	for each in sys.argv[1]:
+		if each == ' ':
+			sys.stdout.write(each)
+		else:
+			sys.stdout.write(chars[each.upper()])
+	sys.stdout.write('\n')
+	return
```
Comment (clafoutis42) [inline chall02/darodrig.py:27]: Could have been written
```suggestion
        print("".join(chars.get(c, " ") for c in sys.argv[1].upper()))
```

**Review by clafoutis42 — COMMENTED** [review]
Good job, you have a working solution, however it is not very pythonic. You are writing your code as if you were using C, I suggest you read [Fluent Python](https://www.amazon.com/Fluent-Python-Concise-Effective-Programming/dp/1491946008) if you want to keep progressing in python!

**Comment by clafoutis42** [comment]
After reflection we won't accept PRs that are not executable as specified in the README.
I have to revert this PR.

**Comment by d-r-e** [comment]
At least I learned something new...
I hope this is taken into account at least, for my coalition, and not completely discarded :(
Thank you.

**Pull request merged**